src/main/webapp/app/shared/virtual-scroll/virtual-scroll.component.ts

```diff
@@ -0,0 +1,261 @@
+/**
+ * Component based on Open Source Project ag-virtual-scroll
+ * https://github.com/ericferreira1992/ag-virtual-scroll
+ *
+ */
+
+import { Component, ElementRef, EventEmitter, Input, OnChanges, OnDestroy, OnInit, Output, Renderer2, SimpleChanges, ViewChild } from '@angular/core';
+import { NavigationStart, Router } from '@angular/router';
+import { VirtualScrollRenderEvent } from 'app/shared/virtual-scroll/virtual-scroll-render-event.class';
+import { ITEMS_PER_PAGE } from 'app/shared/constants/pagination.constants';
+
+@Component({
+    selector: 'virtual-scroll',
```

Comment:
This does not follow our selector naming standard `jhi-`

Reply:
Nice catch!